src/UnitySampleProject/Assets/Resources/HyperPlayConfigData.asset

```diff
@@ -0,0 +1,17 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: c2292ae192d34362ac76a388d530dbb4, type: 3}
+  m_Name: HyperPlayConfigData
+  m_EditorClassIdentifier: 
+  storedSessionAvailable: 0
+  storedWallet: 
+  rememberMe: 1
```

Comment:
small change, this should be 0, to have the same behaviour in all the "Remember me" checkboxes. By default the checkbox should not be selected

Reply:
yeah, still testing, ignore this PR for now <3